translations/component-properties/args/en-us.yaml

```
@@ -0,0 +1,4 @@
+title: Component Arguments
+description: > 
+  In Octane, arguments passed to a component are set on a new `args` property, `this.args`, instead of being set directly on the class instance, `this`.
+  This avoids the possibility of overwriting methods and actions, and makes the distinction between internal class values and arguments much more clear. 
```

Can we change the description to

```suggestion
description: >
  In Octane, arguments are set on the '<code>'args'</code>' property, '<code>'this.args'</code>'.
  They are '<em>'not'</em>' set on the class instance, '<code>'this'</code>'.
  As a result, you can distinguish internal class values from external arguments.
  (Visit the Ember Guides to learn more about '<a href="https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/#toc_arguments" target="_blank" rel="noopener noreferrer">'arguments'</a>'.)
```

In general, I prefer short sentences and not using conjunctions. For the Octane cheatsheet, I think having short sentences can also help with translating to other languages.

---

I'll go ahead with committing my suggestion. Hope that's ok.

---

Yes, I thought I had already done that :-D.